### PR TITLE
Update documentation (ko)

### PR DESCRIPTION
**Pull request by marocchino**
- Up to 4ac9465
- ref #974

### CODE REVIEW

**Comment by marocchino** [comment]
@namsk review please. :bow:

File: ko/documentation/index.md [hunk 1]
```diff
@@ -146,13 +150,11 @@ ruby -v
...
 ### 더 읽을거리
 
-[Ruby-Doc.org][34]은 루비에 관한 문서를 관리하는 사이트입니다. 주로 영문이기는
-하지만 잘 정리된 참조 매뉴얼은 도움이 됩니다.
-
-루비에 대한 [책들][ko-5]도 출판되어 있습니다. 원서들은 [여기][35]에서 확인하세요.
-루비에 관한 질문이 있다면, [유저 그룹][ko-3]이나 [페이스북 그룹][ko-4]에서 하실 수 있고
-영어가 되신다면 [메일링 리스트](/ko/community/mailing-lists/)를 사용하셔도
-됩니다.
+[Ruby-Doc.org][34]과 [Omniref.com][40]은 포괄적으로 루비 영문 문서를 관리하는
```
Comment (stomar) [inline ko/documentation/index.md]: I dont't think this statement is true for Omniref.

Comment (marocchino) [inline ko/documentation/index.md]: I am sorry. I can't understand what `true` means. Can you explain more easily? :sob:

Comment (stomar) [inline ko/documentation/index.md]: Sorry, I meant: I cannot find a list of Ruby reading material on the Omniref page.

Comment (marocchino) [inline ko/documentation/index.md]: Ah, I see. Thank you!

**Comment by marocchino** [comment]
This PR should merge after English docs change. I will close this for a while.

**Comment by stomar** [comment]
@marocchino I just now reverted one of the changed paragraphs. Feel free to amend your branch and reopen the PR.

**Comment by marocchino** [comment]
Apply 6bc1762

**Comment by namsk** [comment]
Thanks @marocchino and @stomar

**Pull request merged**